### ISSUE TITLE
Absolute URL source map support

### ISSUE THREAD

**Issue by laurentgoudet** [gruntjs/grunt-contrib-uglify#202]
Hi guys,

I need to host the source maps on a different host than the sources, for access restriction & CDN configuration purpose. I've forked the repo and added two new options, sourceMapLocation to specify the absolute location of the source maps like "//backend.google.com" and sourceMapRoot to specify the destination folder of the source file (i.e., <%= yeoman.dist %> is the Yeoman world).

Are you guys willing to accept a pull request with that feature? Do you have any suggestion on how I should properly implement that?

Thanks,
Laurent

**Comment by monolithed** [comment]
+1

@jmeas, I think that you do not understand what you are saying, where do you work?
Please tell me at least one major project like the Gmail which would host static files in the root domain?
Only crazy or inexperienced users can afford it!

**Comment by jmeas** [comment]
There's no need to make an ad hominem, @monolithed.

I'm not suggesting that anyone host their assets 'in the root domain.' Rather, I made the (arguably poor) assumption that source maps would only be used in development. Generally, projects I've worked out – whether they've used a CDN or not – would work with local assets only when using source maps. I admit I didn't think about a situation where you would have reporting tools operating on your deployed resources.

I'm willing to review a PR that adds this functionality back in, if either of you are interested in making the pull request.
